kafka/adminapi.go

```diff
@@ -421,30 +421,35 @@ func (a *AdminClient) cConfigResourceToResult(cRes **C.rd_kafka_ConfigResource_t
 }
 
 
+
 // ClusterID returns the cluster ID as reported in broker metadata.
 //
+// Returns an error C.RD_KAFKA_RESP_ERR__TIMED_OUT if the operation times out.
```

Comment:
Use the Go type name, we don't want to expose librdkafka or its types.
See generated_errors.go